pages/natural_ventilation.py

```diff
@@ -74,13 +75,13 @@ def update_layout(si_ip):
         ),
         inputs_tab(tdb_set_min, tdb_set_max, dpt_set),
         dcc.Loading(
-            html.Div(
+            dmc.Box(
                 id=ElementIds.NV_HEATMAP_CHART,
                 style={"marginTop": "1rem"},
             ),
             type="circle",
         ),
-        html.Div(
+        dmc.Box(
```

Comment:
same as previous comment, I do not think we need all these box. I never use them in my code